css-grid-1/implicit-grids/grid-layout-auto-columns-rows.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>CSS Grid Layout Test: auto columns/rows</title>
```

Comment:
Maybe a better title would be: "CSS Grid Layout Test: Support for 'grid-template-columns' and 'grid-template-rows' properties"